app/views/govuk_component/docs.yml

```diff
@@ -427,3 +427,26 @@
       next_page:
         url: "next-page"
         title: "Next page"
+- id: "analytics_meta_tags"
+  name: "Analytics Meta Tags"
+  description: "Meta tags to provide analytics information about the current page"
+  body: |
+    Currently this just takes a content-store links hash like object which it
+    can then turn into the correct analytics identifier metadata tags.
+  fixtures:
+    default: {}
+    with_orgnisations:
+      links_hash:
+        orgnisations:
```

Comment:
There's some typos here. `with_orgnisations`, `orgnisations` and `lead_orgnisations`